resources/todomvc/architecture-examples/preact/src/app/model.js

```diff
@@ -1,13 +1,21 @@
-function uuid() {
-    let uuid = "";
-    for (let i = 0; i < 32; i++) {
-        let random = (Math.random() * 16) | 0;
-        if (i === 8 || i === 12 || i === 16 || i === 20)
-            uuid += "-";
+// Borrowed from https://github.com/ai/nanoid/blob/3.0.2/non-secure/index.js
```

Comment:
We should probably include the LICENSE file with this.

Reply:
added on top of the file.